site/en/models.md

```diff
@@ -0,0 +1,105 @@
+  * [frk](https://github.com/tesseract-ocr/tessdata_fast/raw/master/frk.traineddata)
+  * [script/Latin](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
+  * [script/Fraktur](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
+  * [@stweil's GT4HistOCR model](https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/fast/Fraktur_50000000.334_450937.traineddata)
+
+OCR-D's Tesseract wrapper,
+[ocrd_tesserocr](https://github.com/OCR-D/ocrd_tesserocr) and more
+specifically, the `ocrd-tesserocr-recognize` processor, expects the name of the
+model(s) to be provided as the `model` parameter. Multiple models can be
+combined by concatenating with `+` (which generally improves accuracy but always slows processing):
+
+```sh
+# Use the deu and frk models
+ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESS -p '{"model": "deu+frk"}'
+# Use the script/Fraktur model
+ocrd-tesserocr-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-TESS -p '{"model": "script/Fraktur"}'
+```
+
+## Ocropy / ocrd_cis
+
+An Ocropy model is simply the neural network serialized as with Python's pickle
+mechanism and is generally distributed in a gzipped form, with an `.pyrnn.gz`
+extension.
+
+Ocropy has a rather convoluted algorithm to look up models, so we recommend you
+explicitly set the `OCROPUS_DATA` variable to point to the directory with
+ocropy's models. E.g. if you intend to store your models in `$HOME/ocropus-models`, add the following
+to your `$HOME/.bashrc`: `export OCROPUS_DATA=$HOME/ocropus-models`.
+
+We recommend you download the following models, either by downloading and
+saving to the right location or by running `make install-models-ocropus` when
+using `ocrd_all`:
+
+  * [en-default.pyrnn.gz](https://github.com/zuphilip/ocropy-models/raw/master/en-default.pyrnn.gz)
+  * [fraktur.pyrnn.gz](https://github.com/zuphilip/ocropy-models/raw/master/fraktur.pyrnn.gz)
+  * [@jze's fraktur.pyrnn.gz](https://github.com/jze/ocropus-model_fraktur/raw/master/fraktur.pyrnn.gz) (save as `fraktur-jze.pyrnn.gz`)
+  * [@chreul's  LatinHist.pyrnn.gz](https://github.com/chreul/OCR_Testdata_EarlyPrintedBooks/raw/master/LatinHist-98000.pyrnn.gz)
+
+
+To use a specific model with OCR-D's ocropus wrapper in [ocrd_cis](https://github.com/cisocrgroup/ocrd_cis) and more specifically, the `ocrd-cis-ocropy-recognize` processor, use the `model` parameter:
+
+```sh
+ocrd-cis-ocropy-recognize -I OCR-D-SEG-LINE -O OCR-D-OCR-OCRO -p '{"model": "fraktur-jze.pyrnn.gz"}'
+```
+
+## Calamari / ocrd_calamari
+
+Calamari models are Tensorflow model directories. For distribution, this
+directory is usually packed to a tarball or ZIP file. Once downloaded, these
+containers must be unpacked to a directory again.
+
+As calamari does not have a model discovery setup, you must always provide the
+path with a wildcard listing all `*.ckpt.json` ("checkpoint") files.
```

Comment:
...`absolute path with a literal (unexpanded) wildcard`...

Reply:
It's not necessarily an absolute path. The wildcard could only be picked up by the shell if the JSON expression is completely escaped except for the asterisk.

I.e.

```
ocrd-calamari-recognize -p '{"checkpoint": "model/*.cpkt.json"}'
ocrd-calamari-recognize -p "{\"checkpoint\": \"model/*.cpkt.json\"}"
```

would not trigger the issue, only

```
ocrd-calamari-recognize -p \{\"checkpoint\": \"model/*.cpkt.json\"\}
```

might, but if the user is able to escape all other characters, I'd assume they would also escape the asterisk.

Reply:
AFAICT currently relative path names just won't work, because in the scope of the processor CWD is the workspace directory, which is really hard to relate to package installations.

What I meant by unexpanded wildcard is that the expansion **must** happen in calamari, you cannot pass the _result_ of a regular shell expansion (which would be a space-separated list).